lib/src/id3.ts

```diff
@@ -14,3 +14,4 @@ export { getId3Data } from './id3/getId3Data.js';
 export { getId3Frames } from './id3/getId3Frames.js';
 export { getId3Timestamp } from './id3/getId3Timestamp.js';
 export { isId3TimestampFrame } from './id3/isId3TimestampFrame.js';
+export { decodeId3TextFrame } from './id3/util/decodeId3TextFrame.js';
```

Comment:
Was this export added for a specific reason? Is there a use case where it can be used outside of `getId3Frames`?

Reply:
Assuming we want to maintain the id3 tests within hls.js, we bring in those functions and run the hls.js tests with them. It turns out the decodeId3TextFrame function is needed for one of those tests, which is why we're exporting it.

Reply:
We should remove the ID3 related tests from `hls.js` when we replace their implementation with CML's.

Reply:
Hey @littlespex , regarding those tests, we've discussed that with @nicolaslevy and agreed to left them there so that hls.js devs can make sure that their code works as they expect to, and if changes are made in CML that break their player they can know.

Reply:
If the tests use CML's version of the decode function, they wouldn't necessarily know if things are broken anyway. That type of test would need to be independent of CML. Also, `decodeId3TextFrame` is an internal function which means we do not guarantee its type signature will be stable. Part of the purpose of offloading functionality to CML is to reduce the number of features each player needs to test. We did the same thing when we replaced CMCD/CMSD in `hls.js` and `dash.js`: Once the feature was replaced with CML's version, we removed the tests from the player library.

Reply:
From `hls.js'`s perspective, all they would need to test is that a stream with an ID3 tag produces the appropriate `metadata` text cue point. There is no need to know the internals of the function that created the cue point.

Reply:
Got it, we'll make the changes!

Reply:
fixed!